cornac/models/gp_top/nbr_gp_top.py

```diff
@@ -0,0 +1,76 @@
+# Copyright 2023 The Cornac Authors. All Rights Reserved.
```

Comment:
Let's follow the file naming convention by using `recom_` prefix instead of `nbr_`